src/Microsoft.OData.Client/DictionaryExtensions.cs

```diff
@@ -8,11 +8,12 @@ namespace Microsoft.OData.Client
 {
     using System;
     using System.Collections.Generic;
+    using System.Collections.Concurrent;
     using System.Diagnostics;
     using System.Diagnostics.CodeAnalysis;
-
+    

```

Comment:
nit: remove empty line.

Reply:
removed spaces

---
In reply to: [166142363](https://github.com/OData/odata.net/pull/1054#discussion_r166142363) [](ancestors = 166142363)